src/intl/messages.js

```diff
@@ -584,6 +584,8 @@ export const messages: { [messageId: string]: MessageType } = {
   templateDefined: 'Template Defined',
   timeAgo: '{time} ago',
   timezone: 'Timezone',
+  thereAreNoAvailableClusters: 'There Are No Available Clusters',
+  thereAreNoAvailableCDs: 'There Are No Available CDs',
```

Comment:
What about making the messages shorter? We don't need to see the whole sentence in the drop down :)
My suggestion:
```
noCdsAvailable: 'No CDs available',
noClustersAvailable: 'No Clusters available',
```
Check how it is already done for a VM, for example: https://github.com/oVirt/ovirt-web-ui/blob/master/src/intl/messages.js#L470

Reply:
I second the suggested text.

Side note: Only the first word of the sentence and the actual thing being referenced would be capitalized.  So, if the message would stay the same, it should read "There are no Clusters available".